lightning/src/ln/router.rs

```diff
@@ -955,8 +986,11 @@ impl Router {
 					}
 				}
 
+				log_trace!(self, "Node {} requires {}", $node_id, $node.features.requires_unknown_bits());
```

Comment:
Oof. This s a bit overly verbose. Maybe add them in else clauses so that they aren't always printed?